modules/tools/sensor_calibration/data_file_object.py

```diff
@@ -59,15 +60,32 @@ def __init__(self, file_path, operation='write', file_type='txt'):
                                                   operation, file_type)
 
     def save_to_file(self, data):
-        if not isinstance(data, list):
+        if not isinstance(data, list) and not isinstance(data, np.ndarray):
             raise ValueError("timestamps must be in a list")
 
         for i, ts in enumerate(data):
-            self._file_object.write("%06d %.6f\n" %(i, ts))
+            self._file_object.write("%06d %.6f\n" %(i+1, ts))
 
 class OdometryFileObject(FileObject):
     """class to handle gnss/odometry topic"""
 
+    def load_file(self):
+        struct_len = struct.calcsize('i')
+        data_size = struct.Struct('i').unpack(self._file_object.read(struct.calcsize('i')))[0]
+        print("data_size=%d" % data_size)
+        s0 = struct.Struct('d')
+        s1 = struct.Struct('I')
+        s2 = struct.Struct('7d')
+        data = np.zeros((data_size,9), dtype='float64')
+        #, int32, float64, float64, float64, float64, float64, float64, float64')
```

Comment:
Please fix all comments following:
Start with `#`, then exactly one space, and then the content.